setup.sh

```diff
@@ -14,215 +14,228 @@
 #
 #################################################################################
 
-time=$(date +"%s")
-
 if [[ $EUID -ne 0 ]]; then
     echo "Swizzin setup requires user to be root. su or sudo -s and run again ..."
     exit 1
 fi
 
-if [[ ! $(uname -m) == "x86_64" ]]; then
-    echo -e "\033[0;31mExperimental architecture ($(uname -m)) detected! \033[0m"
-    echo
-    echo "We are in the process of bringing arm support to swizzin. Please let us know on github if you find any issues with a PROPERLY filled out issue template."
-    echo "As such, we cannot guarantee everything works 100%, so please don't feel like you need to speak to the manager when things break. You've been warned."
+export log=/root/logs/swizzin.log
+mkdir -p /root/logs
+touch $log
+
+# Setting up /etc/swizzin
+#shellcheck disable=SC2120
+function _source_setup() {
+    echo "Installing git"              # The one true dependency
+    apt-get install git -y -qq >> $log # DO NOT PUT MORE DEPENDENCIES HERE DASS STUPIT
+    :                                  # All dependencies go to scripts/update/10-dependencies.sh
+
+    if [[ "$*" =~ '--local' ]]; then
+        RelativeScriptPath=$(dirname "${BASH_SOURCE[0]}")
+        if [[ ! -e /etc/swizzin ]]; then # If there is no valid file or dir there...
+            ln -sr "$RelativeScriptPath" /etc/swizzin
+            echo "The directory where the setup script is located is symlinked to /etc/swizzin"
+        else
+            touch /etc/swizzin/.dev.lock
+            echo "/etc/swizzin/.dev.lock created"
+        fi
+        echo "Best of luck and please follow the contribution guidelines cheerio"
+    else
+        echo "Cloning swizzin repo to localhost"
+        git clone https://github.com/swizzin/swizzin.git /etc/swizzin >> ${log} 2>&1
+        echo -e "\tSwizzin cloned!"
+    fi
+    ln -s /etc/swizzin/scripts/ /usr/local/bin/swizzin
+    chmod -R 700 /etc/swizzin/scripts
```

Comment:
Didn't you want to get rid of this line because it causes git to be chatty when doing `box update`s?

Reply:
Not if we keep our permissions in check ha